articles/references/concepts/stereotypes.md

```diff
@@ -0,0 +1,76 @@
+# Stereotypes
+
+## What is a Stereotype?
+
+>Stereotype is a profile class which defines how an existing metaclass may be extended as part of a profile. It enables the use of a platform or domain specific terminology or notation in place of, or in addition to, the ones used for the extended metaclass.
+
+_from: [uml-diagrams.org](https://www.uml-diagrams.org/stereotype.html)_
+
+According to the quote from the UML Diagrams website, a stereotype defines in UML terms that a given Class will have additional meta-data (in a simplified way).
+
+Intent Architect has adopted this notation called `Stereotype` in order to apply additional meta-data on-top of existing Elements.
+
+## What does Stereotypes look like?
+
+Stereotypes are quite flexible in how they appear. You can have a visible indicator on the Element that you work on or it might be only a field where you need to supply additional information in the form of a text field or the like.
+
+![Visible stereotype](images/stereotypes/visible-stereotype.png)
+
+Notice in this image the two downward arrows. That is a stereotype.
+
+![Properties](images/stereotypes/person-class-properties-stereotype.png)
+
+When you inspect the properties of the `Person` Class/Element, you will notice the there is a section called `Role` with the two downward arrows next to it. This is a stereotype that we have applied.
+Also, notice that it contains a filed called `Type` and it has a value that has been selected from a dropdown menu.
+You will begin to understand that Stereotypes allow developers to define their applications in more "domain specific" terms, rather than technical terms.
```

Comment:
```suggestion
Because the stereotype has been applied to it, when you inspect the properties of the `Person` Class/Element, you will notice the there is a section called `Role` (which is the name of the stereotype) with the ____(icon)____ next to it.

Under the `Role` section, there is a field called `Type` and its value has been selected from a dropdown menu.

Stereotypes allow developers to define their applications in more "domain specific" terms, rather than being constrained to default field names in Intent Architect.
```